### PR TITLE
]journal-week-5.md to master

### PR DESCRIPTION
**Pull request by s4dmw**

### CODE REVIEW

**Comment by karjac14** [comment]
that chessboard again! great to know everyone had better grasp with the chessboard. :+1:

**Comment by jeremyrist** [comment]
:+1: glad the chessboard was helpful!

**Comment by drumslayert** [comment]
tutorial is looking sharp :+1: I'm Interested in reading the final.

**Comment by jeffworley** [comment]
Great topics.  Looking forward to the finals. :+1:

**Comment by jeffworley** [comment]
Your reflective journal and tutorial look great.  Didn't see the resource.  Was this a possible late night oversight? :+1:

**Comment by s4dmw** [comment]
@jeffworley it's in there.

**Comment by rosalsm** [comment]
Direct, short and easy to read. :+1:

**Comment by rosalsm** [comment]
Postman is a good resource :+1:

**Comment by jeffworley** [comment]
Update looks solid!  I really like your justification for why readers should use it. :+1:

File: tutorial-week-5.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#### Week 5 Tutorial: [jQuery](http://jquery.com/)
```
Comment (kellymurray) [inline tutorial-week-5.md:1]: Would love to see a little more technical tutorial next time.

**Pull request merged**